packages/integrations/src/vue.ts

```diff
@@ -252,7 +247,7 @@ export class Vue implements Integration {
       }
     };
 
-    const childHandler = (hook: Hook) => {
+    const childHandler = (hook: Hook, operation: Operation) => {
```

Comment:
Is there an option we expose to make this per component? Like if a user wanted to track multiple hooks for a single component, but use the standard `hooks: ['mount', 'update']` for everything else.

Reply:
No, it's all configured for the whole app right now. We might do this in the future though.